### DISCUSSION TITLE
Documentation to interact with the supabase without the client library?

### DISCUSSION THREAD

**Discussion by nivekithan** [supabase/supabase#1190] [Questions] [answered]
Hi,  is there any documentation on dealing with `supabase` directly without any library. Like `rest` api or something like that?

**Comment by gomflo** [comment]
supabase database is built on top of PostgREST, maybe you will find this useful: https://supabase.io/docs/postgrest/client/postgrest-client

**Reply by nivekithan** [reply]
I am not really sure how that is helpful since it seems the documentation is for `@supabase/postgrest-js`

**Reply by gomflo** [reply]
i see, then you can connect directly to the database using the connection info in settings -> database 

![image](https://user-images.githubusercontent.com/141670/115152683-ab266e00-a037-11eb-9ea0-81a3473e16b7.png)

**Reply by nivekithan** [reply]
Hey thanks for the help. But what I am looking for is something like [this](https://github.com/supabase/supabase/discussions/860#discussioncomment-473516) which includes every single parameter and such

**Comment by inian** [comment, marked as answer]
For postgrest, here are the [docs](https://postgrest.org/en/stable/api.html). The endpoint to use is `https://projectref.supabase.co/rest/v1`. 

For storage, the docs are [here](https://supabase.github.io/storage-api/#/). The endpoint to use is `https://projectref.supabase.co/storage/v1`

**Reply by nivekithan** [reply]
Thanks this is exactly what I needed 👍

**Reply by kiwicopple** [reply]
You can also find the cURL endpoints inside the "Docs" section of your project (https://supabase.io/docs/guides/api#accessing-the-docs)

**Reply by Splizard** [reply]
The storage API docs link is dead

**Comment by cosmic-zip** [comment]
some links are dead, mainly the link about the storage.

dead links:
https://supabase.github.io/storage-api/#/
https://supabase.com/docs/postgrest/client/postgrest-client

working link:
https://postgrest.org/en/stable/references/api.html

**Reply by GaryAustin1** [reply]
> some links are dead, mainly the link about the storage.

https://supabase.github.io/storage/

Also Auth:  https://github.com/supabase/auth?tab=readme-ov-file#endpoints

The dashboard also has a useful API docs tab from similar stuff.
